README.md

```diff
@@ -142,9 +142,10 @@ See [4-4-5 Calendar](https://en.wikipedia.org/wiki/4%E2%80%934%E2%80%935_calenda
 
 #### LastMonthOfYear
 
-Specifies the month the year ends. An integer from 0 (January), to 11 (December). Currently last month of retail year is always assumed to be in next calendar year.
+Specifies the month the year ends. An integer from 0 (January), to 11 (December). Last month of retail year is assumed to be in next calendar year except for December.
```

Comment:
@cihancimen I think this is confusing. Technically, this is correct but I don't think people think this way. If you want to refer to year selection, we can explain it here or somewhere else but I think we should say something like, 'Plan year is the Gregorian year of the first month.'